composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt

```diff
@@ -45,11 +58,54 @@ fun App(dao: DemoDao) {
     }
 
     nfcManager.startScan()
-    
+
     MRTBuddyTheme {
-        MainScreen(
-            cardState = McardState.value,
-            transactions = Mtransactions.value
-        )
+        var lang by remember { mutableStateOf(Language.English.isoFormat) }
+        LocalizedApp (
+            language= lang
+        ) {
+            Scaffold {
+                Box(
+                   Modifier.systemBarsPadding()
+                ) {
+                    Column {
+                        TopAppBar(
+                            title = {
+                                Text(text = stringResource(Res.string.appName))
+                            },
+                            backgroundColor = MaterialTheme.colors.background,
+                            actions = {
+                                OutlinedButton(
+                                    onClick = {
+                                        lang = switchLanguage(lang)
+                                        changeLang(lang)
+                                    },
+                                    shape = RoundedCornerShape(24.dp),
+                                ) {
+                                    Text(text = stringResource(Res.string.language))
```

Comment:
Perhaps the text should be the name of the Other language if we are in a language toggle situation. Like the guy who doesn't know English cannot read the word English (ok very hypothetical and)